packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 22.8.0
+
+* [swift, kotlin] Added support of modern asynchronous api for Swift (async) and Kotlin (suspend).
```

Comment:
i'd just call it "swift concurrency and kotlin coroutines"

Reply:
Changed to "[swift, kotlin] Added support of swift concurrency and kotlin coroutines."